client/state/plugins/installed/selectors.js

```diff
@@ -0,0 +1,164 @@
+import filter from 'lodash/filter';
+import forEach from 'lodash/forEach';
+import values from 'lodash/values';
+import sortBy from 'lodash/sortBy';
+import find from 'lodash/find';
+import uniq from 'lodash/uniq';
+import compact from 'lodash/compact';
+import omit from 'lodash/omit';
+import every from 'lodash/every';
+import some from 'lodash/some';
+
+const _filters = {
+	none: function() {
+		return false;
+	},
+	all: function() {
+		return true;
+	},
+	active: function( plugin ) {
+		return plugin.sites.some( function( site ) {
+			return site.plugin && site.plugin.active;
+		} );
+	},
+	inactive: function( plugin ) {
+		return plugin.sites.some( function( site ) {
+			return site.plugin && ! site.plugin.active;
+		} );
+	},
+	updates: function( plugin ) {
+		return plugin.sites.some( function( site ) {
+			return site.plugin && site.plugin.update && site.canUpdateFiles;
+		} );
+	},
+	isEqual: function( pluginSlug, plugin ) {
+		return plugin.slug === pluginSlug;
+	}
+};
+
+const isRequesting = function( state, siteId ) {
+	// if the `isRequesting` attribute doesn't exist yet,
+	// we assume we are still launching the fetch action, so it's true
```

Comment:
I agree with @aduth on this. I think we should return `false` in that case, because the request hasn't started. 

If you are still in doubt, have a look at the `QueryPlugins` component (it's still in progress, but anyway). If you look [here](https://github.com/Automattic/wp-calypso/blob/1ed9ca66f54a926f70f157d972b71ccc20e0043e/client/components/data/query-plugins/index.jsx#L17) you will notice that the initial fetch will never occur, since `this.props.isRequestingForSites` is initially `true`.